docs/modules/ROOT/pages/vshn-managed/postgresql/maintenance.adoc

```diff
@@ -0,0 +1,22 @@
+= Scheduled Maintenance
+
+By default, there will be a security maintenance on Tuesday night.
```

Comment:
Can we be more specific what "security maintenance" means? I'd also like to have a detailed description what happens during maintenance, this is seemingly overly scarce here...

Reply:
would a link to https://products.docs.vshn.ch/products/appcat/redis.html#_recurring_maintenance be sufficient?

I was intentionally not specific as it's already pretty specific there and want to avoid duplicate information, that could drift apart...

Reply:
Yes, go for it